lang/strings_english.txt

```diff
@@ -0,0 +1,14 @@
+<?php
+# Copyright (c) MantisBT Team - mantisbt-dev@lists.sourceforge.net
+# Licensed under the MIT license
+
+$s_plugin_SimpleSAMLphpAuth_title = 'AuthSimpleSAMLphp';

```

Comment:
This name is not consistent with the plugin name.  I also think it would make more sense to use a more user friendly name here.  For example, `SAML Auth` or `SAML Auth via SimpleSAMLAuth`.